### DISCUSSION TITLE
bug: gnutls_handshake() failed: The TLS connection was non-properly terminated

### DISCUSSION THREAD

**Discussion by makinzm** [LazyVim/LazyVim#4995] [Q&A] [answered]
### Did you check docs and existing issues?

- [x] I have read all the LazyVim docs
- [x] I have updated the plugin to the latest version before submitting this issue
- [x] I have searched the existing issues of LazyVim
- [x] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

NVIM v0.10.2

### Operating system/version

Ubuntu 22.04

### Describe the bug

I was trying to add a Package from Lazyvim for the first time in a while and encountered the following bug.

`gnutls_handshake() failed: The TLS connection was non-properly terminated.`
![Image](https://github.com/user-attachments/assets/a0744fbc-df15-4f22-b737-16fb904c7493)

I had made the following changes since the most recent time I successfully added Package from Lazyvim, but I do not remember the details.

However, I do remember some of them and will list them as best I can.

In the meantime, I made some changes to the DNS server address and did the following Package Install

````
sudo apt-get update
sudo apt-get install -y build-essential r-base-dev libssl-dev libcurl4-openssl-dev libxml2-dev
sudo apt-get install -y libblas-dev liblapack-dev gfortran libeigen3-dev
````

The error seems to be related to the git configuration, but as long as I type git from the local environment, there is no problem, and the error is occurring only in Neovim and LazyVim.

### Steps To Reproduce

1. git clone the following repository to ~/.config/nvim i.e. `git clone git@github.com:makinzm/lazyvim.git nvim` on `~/.config`
    -  [makinzm/lazyvim](https://github.com/makinzm/lazyvim)
2. Update ssl tools like ( it may be insufficient )
    ```shell
    sudo apt-get update
    sudo apt-get install -y build-essential r-base-dev libssl-dev libcurl4-openssl-dev libxml2-dev
    sudo apt-get install -y libblas-dev liblapack-dev gfortran libeigen3-dev
    ```
4. command `nvim` and then the error page occured.

### Expected Behavior

no error occurs.

### Repro

```lua
-- bootstrap lazy.nvim, LazyVim and your plugins
require("config.lazy")

-- Example: Add this to your init.lua or a sourced Lua file
vim.api.nvim_set_hl(0, "LineNr", { fg = "#cccc00" }) -- Change to a bright color like yellow
vim.api.nvim_set_hl(0, "CursorLineNr", { fg = "#00ff00", bold = true }) -- Make the current line number green and bold

-- Disable copilot in defaults
vim.cmd(":Copilot disable")

vim.cmd("set conceallevel=0")
```

**Comment by folke** [comment, marked as answer]
check your network. check your OS settings. Look for similar issues here and probably in the lazy.nvim repo.
Not a LazyVim, nor lazy.nvim bug.

**Reply by makinzm** [reply]
@folke Thanks, so I'll check it.

However, this problem occurred in only neovim/lazyVim but didn't occur in shell(zsh).
How were you able to identify it as an OS or Network issue?

**Reply by makinzm** [reply]
@folke After I reset my network machine, the error disappear. Thank you :bow:
